examples/practice-typescript/src/components/SideBarMenuItem/sideBarMenuItem.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../styles/bases/variable.scss';
+
+.aside__item {
+  flex-direction: column;
+  justify-content: center;
+  height: 60px;
+  text-decoration: none;
+  display: flex;
+  font-family: Roboto;
+  font-weight: 400;
+  color: $color-text-secondary;
+}
+
+.aside__item {
+  width: 100%;
+  border-right: 3px solid $color-bg-manager;
+  font-weight: 400;
+}
+
+.aside__active {
+  border-right: 3px solid $color-active-secondary;
```

Comment:
should be border-right-color

Reply:
Fixed

Reply:
@thanhtrantruong47 This one is not resolved yet